packages/d3fc-webgl/src/symbolMapper.js

```diff
@@ -0,0 +1,25 @@
+import d3Shape from 'd3-shape';
+import circlePointShader from './shaders/point/circle/baseShader';
+import squarePointShader from './shaders/point/square/baseShader';
+
+export default (symbol) => {
+  let outSymbol = d3Shape.symbolCircle;
```

Comment:
I believe the `outSymbol` equivalent in the scale mapper is used in combination with the `glSymbol` equivalent. However, there's no way of combining the two symbols (happy to be proved wrong!) so I think we should remove `outSymbol` and just return `glSymbol` (without the object wrapper).